packages/waku/src/router/client.ts

```diff
@@ -551,6 +551,14 @@ export function ServerRouter({
   );
 }
 
+function renderError(message: string) {
+  return createElement(
+    'html',
+    null,
+    createElement('body', null, createElement('h1', null, message)),
+  );
+}
```

Comment:
This doesn't feel like a final solution, but it looks good to me for now at least.